### PR TITLE
Added Support for latest version of Hoppscotch UI

### PR DESCRIPTION
**Pull request by Suraj-Tiwari**
- added support for CONFIG_DIR_PATH env
- added postgres support
- added missing graphql handlers for latest version of hoppscotch
- added docker-compose
- did minor cleanup and fixes 
- updated readme

I've tried to keep as close as possible to existing design pattern, if there are any changes needed you can let me know.

### CODE REVIEW

File: config.example.yaml [hunk 1]
```diff
@@ -5,8 +5,10 @@ api:
 database:
   username: "hoppscotch"
   password: "hoppscotch"
-  address: "tcp(127.0.0.1:3306)"
   database: "hoppscotch"
+  host: "127.0.0.1"
```
Comment (jerbob92) [inline config.example.yaml:9]: Could you add some fallback code for the people that still have the old `address` config?

**Review by jerbob92 — COMMENTED** [review]
Thanks a lot! Looks good :) 
Just one minor note for backwards compatibility.

File: config.example.yaml (hunk 1 above)
Comment (Suraj-Tiwari) [reply config.example.yaml:9]: sure

**Comment by Suraj-Tiwari** [comment]
@jerbob92 please have a look, added backward compatibility with deprecated message

File: models/shortcode.go [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 	gorm.Model
-	Code    string
-	Request string
-	UserID  uint
-	User    User
+	Code      string
+	Request   string
+	UserID    uint
+	User      User
+	CreatedOn time.Time `gorm:"autoCreateTime"`
```
Comment (jerbob92) [inline models/shortcode.go:14]: Why not use the default `CreatedAt` of the `gorm.Model` for this?

**Comment by jerbob92** [comment]
> @jerbob92 please have a look, added backward compatibility with deprecated message

Great, thanks! I have one more minor comment.

File: models/shortcode.go (hunk 2 above)
Comment (jerbob92) [inline models/shortcode.go]: Why not use the default `CreatedAt` of the `gorm.Model` for this?

Comment (Suraj-Tiwari) [reply models/shortcode.go:14]: just though on having 1 on 1 mapping with graphql as wasn't able to figure out conviction for timing for this specific project, you're right as in general it's CreatedAt or UpdatedAt.

can update if you want.

Comment (Suraj-Tiwari) [reply models/shortcode.go]: Just checked docs, my bad i haven't worked with gorm before. i'll update it right away

**Comment by Suraj-Tiwari** [comment]
@jerbob92 updated

**Comment by jerbob92** [comment]
Thanks a lot for your contribution @Suraj-Tiwari! :partying_face:

**Pull request merged**